### ISSUE TITLE
Inconsistent icon elements

### ISSUE THREAD

**Issue by marksreeves** [BorderTech/wcomponents#1497]
We have converted most fo the icons to use an `i` element and ensure they are all aria-hidden. Some (especially in templates) still use `span` elements. This should be made consistent.
